docs/instant.md

```diff
@@ -626,8 +627,9 @@ Example usage:
 
 ```js
 instant = Temporal.Instant.from('2019-11-18T11:00:00.000Z');
-instant.toLocaleString(); // => example output: 2019-11-18, 3:00:00 a.m.
-instant.toLocaleString('de-DE'); // => example output: 18.11.2019, 03:00:00
+instant.toLocaleString(); // => '11/18/2019, 3:00:00 AM'
```

Comment:
I prefer the "example output" phrasing because even in the same locale the output is not guaranteed to be the same across browsers.

Reply:
ah, okay, I removed it for automation; I'll add it back in.